### DISCUSSION TITLE
Inline or Quote Comments

### DISCUSSION THREAD

**Discussion by cocoahero** [community/community#3566] [Discussions]
One thing that I am missing from the pull request UX is the ability to leave inline comments. Perhaps this is not the goal of the Discussions feature, but we're experimenting with using Discussions for longer-format proposals. Having the ability to comment inline on particular sentences would be nice. Or, perhaps an ability to highlight some text, right click and say "quote" reply.

**Comment by luispadron** [comment]
I would love to see something like this as well. The ability to highlight certain parts of a larger text is super important. The UI for PR comments where you can highlight lines in the PR and comment works great. 

Something like that or even just nicer support for quoting / replying to parts of text will really help when dealing with long form content like RFC / proposals.

**Comment by coury-clark** [comment]
Echoing this - discussions are a more useful tool for long-form feedback and would benefit greatly from the PR UI to comment / suggest inline.
